.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  repository_dispatch:
+    types: [release]
+  schedule:
+    - cron:  '0 0 15 * *'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: "Release a new version"
+    steps:
+        - id: calver
+          if: ${{ !github.event.client_payload.version }}
+          run: echo "::set-output name=version::$(date +'%y.%-m.0')"
+        - uses: actions/checkout@v2
+        - uses: getsentry/craft-action@master
+          with:
+            action: prepare
+            version: ${{ github.event.client_payload.version || steps.calver.outputs.version }}
+            dry_run: ${{ github.event.client_payload.dry_run }}
```

Comment:
For my education: How would the dry run be set?

Reply:
This is what I've been using to test this:
```sh
curl -X POST -u "BYK:<redacted_github_pat>" -H "Accept: application/vnd.github.everest-preview+json"
 -H "Content-Type: application/json" --data '{"branch": "byk/build/calver-gha
", "event_type":"release", "client_payload": {"dry_run": true}}' https://api.
github.com/repos/getsentry/onpremise/dispatches
```

Reply:
This is obviously for testing getsentry/onpremise.

Reply:
Alright, I had expected something like that. Thanks!